docs/dry-run-mode.md

```diff
@@ -1,35 +1,32 @@
 # Dry Run Mode
 
-As your test suits get bigger and complicated over the period of time, it is essential that the toolings used for creating tests provide an easy way to identify and list the tests being
-processed as part of your framework when invoked with certain arguments. And this listing needs
-to be quick and clean in order to enable quick turn around time of test development. This requirement bring in the need to introduce a `dry-run` behavior into the `e2e-framework`.
+As your test suites get bigger and more complicated over time, it is essential that the tools used for creating tests provide an easy way to identify and list the tests being processed as part of your framework when invoked with certain arguments.
+This listing needs to be quick and clean to enable a quick turnaround time for test development. This requirement brings in the need to introduce a `dry-run` behavior into the `e2e-framework`.
```

Comment:
Isn't it usually better to rely on the text viewer to handle line breaks instead of doing one manually in the middle of a sentence for markdown files? This way, the text won't be weirdly cut in the middle like it is right now 
![image](https://github.com/kubernetes-sigs/e2e-framework/assets/57704682/b7e80663-0a6d-45db-969f-e121805c6478)

Reply:
but that will not show up in the markdown, we usually break the line around 80col to fit better in the IDE, but that does not reflect in the actual rendered page as you can see in https://github.com/kubernetes-sigs/e2e-framework/blob/main/docs/dry-run-mode.md

Reply:
Well, isn't it better to rely on word wrapping in the IDE then? 80 cols in only relevant when doing something like half screen on a ~15'' monitor isn't it? So for all other cases this results in lines being split in the middle of the screen
![image](https://github.com/kubernetes-sigs/e2e-framework/assets/57704682/ac3edd30-1379-4e2a-995c-d56f80ce10c1)

Btw, the current text isn't even split at the 80 col mark. It looks like this originally on a 80col screen 
(with word wrap at 80col)
![image](https://github.com/kubernetes-sigs/e2e-framework/assets/57704682/bd8ac911-3b10-470c-80e0-12b4aa8d13cb)
(without word wrap)
![image](https://github.com/kubernetes-sigs/e2e-framework/assets/57704682/c5ad6ea7-1497-4f57-9f90-e14ce75ded56)

While it looks like this now (with word wrap at 80col)
![image](https://github.com/kubernetes-sigs/e2e-framework/assets/57704682/afbf9e35-0159-48ba-8750-32a88bf82d9c)

Reply:
@cpanato is this change a no-go for you? If it is, I can revert to how it was previously

Reply:
that is fine, i guess, but to be honest i don't see the value that adds

Reply:
I would just remove it, keep the git history clean if we're not changing wording